src/components/LanguageButton/LanguageButton.jsx

```diff
@@ -0,0 +1,82 @@
+import { Fragment } from 'react';
+import { Menu, Transition } from '@headlessui/react';
+import { SiGoogletranslate } from 'react-icons/si';
+
+import UsFlag from '../../assets/img/us-flag.svg';
+import IraqFlag from '../../assets/img/iraq-flag.svg';
+
+const classNames = (...classes) => {
+  return classes.filter(Boolean).join(' ');
+};
+
+const LanguageButton = () => {
+  const handleLanguageChange = (language) => {
+    localStorage.setItem('language', language);
+  };
+
+  return (
+    <Menu
+      as="div"
+      data-testid="language-button"
+      className="relative inline-block text-left"
+    >
+      <div>
+        <Menu.Button
+          data-testid="language-botton-toggle"
+          className="md:ml-6 text-2xl md:mt-0 mt-2 text-background hover:text-secondary duration-500"
+        >
+          <SiGoogletranslate />
```

Comment:
what if you can make the lang button the selected flag instead of the translate icon?

Reply:
i have the same idea in the beginning but it doesn't make any change because when the user look at the website the selected language will apply automatically, but we can update it in the future when you add the functionality to the website